ui/src/experiments/list/search/components/ExperimentStatusFilter.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import { EuiComboBox, EuiFormRow, EuiHealth } from "@elastic/eui";
+
+const ExperimentStatusFilter = ({
+  value,
+  options,
+  onChange,
+}) => {
+  const renderOption = o => (<EuiHealth color={o.color}>{o.label}</EuiHealth>);
+  const selectedOptions = options.filter(o => o.value === value);
+  // Filter only fields recognized by the combobox.
+  const optionsFiltered = options.map(o => ({ value: o.value, label: o.label, color: o.color }));
+
+  return (
+    <EuiFormRow fullWidth label={"Experiment Status"}>
+      <EuiComboBox
+        placeholder="Select option"
+        options={optionsFiltered}
+        renderOption={renderOption}
+        selectedOptions={selectedOptions}
+        onChange={e => onChange(e.length > 0 ? e[0].value : undefined)}
+        isClearable={true}
+        data-test-subj={`experiment-status-combo-box`}
+        singleSelection={true}
```

Comment:
Hmm is there a reason why we can't allow multiple selections? 🤔

Reply:
Originally I thought it made more sense to keep this filter single-select. But I guess searching for scheduled + running could be useful. This change will require the API to be compatible with multiple search values as well and it seems there are some issues getting it to work with the current version of Gorm we are using. Let me open a new PR for that, to keep it focussed.